public/app/features/browse-dashboards/components/BrowseActions/BulkMoveFailedBanner.tsx

```diff
@@ -0,0 +1,51 @@
+import { Trans, t } from '@grafana/i18n';
+import { Alert } from '@grafana/ui';
+
+import { BulkMoveResult } from './utils';
+
+// TODO: Adjust this for bulk delete operation
+export function BulkMoveFailedBanner({ result, onDismiss }: { result: BulkMoveResult; onDismiss: () => void }) {
+  if (result.failed.length === 0) {
+    return null;
+  }
```

Comment:
Could/should this check be outside of this component? i.e. if you're rendering the `BulkMoveFailedBanner`, you do so under the assumption that something is always going to display, rather than sometimes nothing showing